fili-core/src/main/java/com/yahoo/bard/webservice/data/metric/mappers/RowNumMapper.java

```diff
@@ -27,35 +26,30 @@
     @Override
     public ResultSet map(ResultSet resultSet) {
 
-        Schema modifiedSchema = map(resultSet.getSchema());
-        MetricColumn metricColumn = modifiedSchema.getColumn(ROW_NUM_COLUMN_NAME, MetricColumn.class);
-
-        List<Result> newResults = new ArrayList<>();
-        Result newResult;
+        ResultSetSchema schema = map(resultSet.getSchema());
+        MetricColumn column = schema.getColumn(ROW_NUM_COLUMN_NAME, MetricColumn.class).orElseThrow(
+                () -> new IllegalStateException("Unexpected missing column")
```

Comment:
I'm just going to remove this check.  We literally just added this column to the schema the line before.